.github/workflows/security-testing.yml

```
@@ -0,0 +1,1048 @@
+  pull_request:
+    branches: [main, develop]
+  schedule:
+    # Run security tests every day at 1 AM UTC
+    - cron: '0 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  authentication-testing:
+    name: Authentication Security Testing
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)